.github/workflows/docker_build_push.sh

```diff
@@ -50,12 +50,39 @@ docker build --target lean \
   -t "${REPO_NAME}:${SHA}" \
   -t "${REPO_NAME}:${REFSPEC}" \
   -t "${REPO_NAME}:${LATEST_TAG}" \
+  --build-arg PY_VER="3.8.16"\
   --label "sha=${SHA}" \
   --label "built_at=$(date)" \
   --label "target=lean" \
   --label "build_actor=${GITHUB_ACTOR}" \
   .
 
+# Build the "lean39" image
+docker build --target lean \
+  -t "${REPO_NAME}:${SHA}-py39" \
+  -t "${REPO_NAME}:${REFSPEC}-py39" \
+  -t "${REPO_NAME}:${LATEST_TAG}-py39" \
+  --build-arg PY_VER="3.9.16"\
+  --label "sha=${SHA}" \
+  --label "built_at=$(date)" \
+  --label "target=lean39" \
+  --label "build_actor=${GITHUB_ACTOR}" \
+  .
+
+
+# Build the "lean310" image
+docker build --target lean \
```

Comment:
Let's only go with `3.9` for now as `3.10` isn't tested yet

Reply:
removed 3.10